Packages/Sources/RootView/TabItem.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import SwiftUI
+
+struct TabItem: View {
+  var tab: Tab
+  @Binding var activeTab: Tab
+  @Environment(\.colorScheme) var colorScheme
+
+  var body: some View {
+    VStack(alignment: .center, spacing: 2) {
+      Image(tab.systemImage)
+        .font(.title2)
+        .frame(width: 24, height: 24)
+        .frame(maxWidth: .infinity)
+        .onTapGesture {
+          activeTab = tab
+        }
+        Circle()
+            .fill(
+                self.tab == activeTab ?
+                (colorScheme == .light ? Color.black : Color.neutral8) : Color.clear)
```

Comment:
`colorScheme == .light ? Color.black : Color.neutral8` This code I guess might get repeated along the code base. For consistency, it might be good to start working on a color scheme for your app. You can take inspiration in apple's naming categories, you can do something similar https://developer.apple.com/documentation/uikit/uicolor/ui_element_colors

Reply:
Indeed, the current naming convention for the colors is obtained from the Figma and is not ideal. 
They are simply labeled as "Primary 1" through "Primary 4," "Secondary 1" through "Secondary 4," and "Neutral 1" through "Neutral 8."
Do you suggest changing the name? 

<img width="285" alt="Screenshot 2024-01-20 at 1 17 32 PM" src="https://github.com/hilalalhakani/NodeCrypto/assets/24891448/414f490c-ae10-4341-9fe3-aa2a5403ab67">